core/api/service/system/requests/version.cpp

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "api/service/system/requests/version.hpp"
+
+namespace kagome::api::system::request {
+
+  Version::Version(std::shared_ptr<SystemApi> api) : api_(std::move(api)) {
+    BOOST_ASSERT(api_ != nullptr);
+  }
+
+  outcome::result<void> Version::init(
+      const jsonrpc::Request::Parameters &params) {
+    if (!params.empty()) {
+      throw jsonrpc::InvalidParametersFault("Method should not have params");
+    }
+    return outcome::success();
+  }
+
+  outcome::result<std::string> Version::execute() {
+    // TODO(xDimon): Ensure if implementation is correct, and remove exception
+    throw jsonrpc::InternalErrorFault(
+        "Internal error: method is known, but not yet implemented");
+
+    return api_->getConfig()->id();
```

Comment:
As explained [here](https://github.com/w3f/PSPs/blob/psp-rpc-api/psp-002.md#system_version) version should be a semver string whereas `id`can be arbitrary string. I think it makes sense to just return 0.0.1 for now